### PR TITLE
Add Dropdown to help in navigation with knowledge base link

### PR DESCRIPTION
**Pull request by astupak**
http://snpy.in/vJA95m

### CODE REVIEW

File: src/kit/page/navigation.ejs [hunk 1]
```diff
@@ -1,98 +1,127 @@
...
+                        direction: 'br'
+                    }
+                }) %>
+            </div>
+
+            <%
+            var userId = require('app').userId || '',
+                    userName = require('app').userName || '';
+            %>
+            <%= this.include(UI.accountDropdown, {
```
Comment (rosendi) [inline src/kit/page/navigation.ejs]: Please, don't use the global UI.

Comment (rosendi) [inline src/kit/page/navigation.ejs]: Use `require` instead.

File: src/kit/page/navigation.ejs [hunk 2]
```diff
@@ -1,98 +1,127 @@
...
-						text: i18n('Автоматизация')
-					}
-				],
-				modifiers: {
-					type: 'primary'
-				}
-			}) %>
-		</div>
+        <div class="header__navigation">
+            <%= this.include(require('blocks/navigation/navigation'), {
```
Comment (rosendi) [inline src/kit/page/navigation.ejs]: please move the require to the top

File: src/kit/page/navigation.ejs [hunk 3]
```diff
@@ -1,98 +1,127 @@
...
-			<% } %>
-			<a href="/track" class="iconBox iconBox_color_white iconBox_size_small">
-				<%= UI.icons.notifications %>
-			</a>
+            <% 
+            var noconfirmRest = this.models.userLimits.get('member@noconfirm@rest');
+            var hardRest = this.models.userLimits.get('member@hard@rest');
+            noconfirmRest = isFinite(noconfirmRest) && noconfirmRest !== null ? +noconfirmRest : Number.POSITIVE_INFINITY;
+            hardRest = isFinite(hardRest) && hardRest !== null ? +hardRest : Number.POSITIVE_INFINITY; 
+            var rate = require('app').tariff;
```
Comment (rosendi) [inline src/kit/page/navigation.ejs]: that's wierd

File: src/kit/page/navigation.ejs (hunk 1 above)
Comment (astupak) [inline src/kit/page/navigation.ejs]: Fixed

**Comment by rosendi** [comment]
LGTM

**Comment by trikashny** [comment]
LGTM

**Pull request merged**